frontend/src/components/AgencyProfile/Contacts.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import edit from "./imgs/edit-icon.png";
+import primaryTab from "./imgs/contact-tab-primary.png";
+import secondaryTab from "./imgs/contact-tab-secondary.png";
+import AgencyBar from "./AgencyBar";
+
+/**
+ * Functional component for the contacts category
+ *
+ * @param {*} agency
+ * @returns {*} Contacts component
+ */
+function Contacts({ agency }) {
+  let secondContact =
+    agency.contacts.length > 1 ? agency.contacts[1] : agency.contacts[0];
```

Comment:
Are we supposed to display the same contact twice if there is only one contact? What if there are more than 2 contacts?

Reply:
I'd suggest using a `map()` call when rendering since the contact boxes look the same. Not sure how the border colors work, perhaps just use the blue for the first, pink for the second (if there), and black for any subsequent ones?

Reply:
`map()` is a good idea, we need to be able to handle more than 2 contacts being added

Reply:
If there was a third contact, would I add another box with a tag "Third Contact"? Or would I just display the primary and secondary contact regardless of the amount of contacts.

Reply:
Ultimately this would be up to Sara/the designers, but I guess you could just say "Contact 3", "Contact 4", and so on for any contacts after "Secondary" - just based on the index in the `map()`. To handle an arbitrary number of contacts you might need to use flexbox and especially flex-wrap for the CSS.

Reply:
I can double check with the designers on what they what contacts past the secondary contact to be called,. but for now I would just go with something like 'Additional' or 'Other' that way there's no need to even do any logic for the naming. I'll see what the designers say and let you know the final decision.

Reply:
Let's go with 'Other' as the way to denote additional contacts past 'Primary' and 'Secondary'